src/main.zig

```diff
@@ -1973,7 +1969,7 @@ pub const Editor = struct {
     }
 
     pub fn insertCodePoint(self: *Self, code_point: u32) void {
-        var buf = [_]u8{ 0, 0, 0, 0 };
+        var buf: [4]u8 = @splat(0);
```

Comment:
this is great! is this `@splat` new?

Reply:
It existed for a long time for `@Vector()` (SIMD) but got extended to arrays in 0.14: https://ziglang.org/download/0.14.0/release-notes.html#splat-Supports-Arrays